src/DotPulsar/Abstractions/IConsumerBuilder.cs

```diff
@@ -83,6 +83,17 @@ public interface IConsumerBuilder
         /// </summary>
         IConsumerBuilder NegativeAcknowledgeRedeliveryDelay(TimeSpan delay);
 
+        /// <summary>
+        /// If true then we load one message from partition at a time, otherwise we load up to NumberOfMessagesToReadFromPartition
+        /// from one partition (reading only messages which are already in the queue, no waiting) before moving to the next.
+        /// </summary>
+        IConsumerBuilder OneMessageFromPartitionAtATime(bool oneMessageFromPartitionAtATime);
```

Comment:
Any reason to have another bool setting when defaulting the other setting to `1` accomplishes the same feat?  Even if we want to maintain completely separate code paths, I'd argue testing the other setting `== 1` would be cleaner than the two settings?

Reply:
fixed